Gruntfile.js

```diff
@@ -120,7 +114,7 @@ module.exports = function(grunt){
   });
 
   [
-    'grunt-contrib-copy',
+    'grunt-sync',
     'grunt-contrib-watch',
     'grunt-contrib-clean',
```

Comment:
Yes, as far as I understand it the way grunt-sync works would remove the need for the initial clean.